packages/terra-date-picker/src/DateInput.jsx

```diff
@@ -794,7 +829,7 @@ const DatePickerInput = (props) => {
         <Button
           data-terra-open-calendar-button
           className={buttonClasses}
-          text={intl.formatMessage({ id: 'Terra.datePicker.openCalendar' })}
+          text={`${calendarDate} ${intl.formatMessage({ id: 'Terra.datePicker.openCalendar' })}`}
```

Comment:
this should be set as aria-label instead to prevent tooltip from showing the selected date along with text `open calendar` when hover on button

Reply:
updated in https://github.com/cerner/terra-framework/pull/1774/commits/67e9043fe4a041fcc8573ca4f0cb82cc709ae204

Reply:
after above commit was this change tested on JAWS

Reply:
Tested after changes